Rocket.Chat/Controllers/Chat/ChatControllerReplyHandler.swift

```diff
@@ -15,7 +15,7 @@ extension ChatViewController {
         replyView.backgroundColor = textInputbar.addonContentView.backgroundColor
         replyView.frame = textInputbar.addonContentView.bounds
         replyView.onClose = stopReplying
-
+        replyView.alpha = 0
```

Comment:
Why did you add this line?

Reply:
Hum this was needed before I started using SlackTextViewController's EditMode (as I had an editView and a replyView and had to manage their visibilities). It isn't needed now, good catch!